convert2rhel/unit_tests/utils_test.py

```diff
@@ -50,6 +51,17 @@ def __call__(self, cmd, print_cmd=True, print_output=True):
             self.called += 1
             return self.output, self.ret_code
 
+    class DummyGetUID(unit_tests.MockFunction):
+        def __call__(self, *args, **kargs):
+            return 1
+
+    @unit_tests.mock(os, "geteuid", DummyGetUID())
+    def test_require_root_is_not_root(self):
+        self.assertRaises(SystemExit, utils.require_root)
+
+    def test_require_root_isroot(self):
```

Comment:
Add a mock for `geteuid` on this one as well, relying on the UID of whichever is running the tests means it's a flaky test

Also for consistency:
```suggestion
    def test_require_root_is_root(self):
```